spec/prodecuralSpec.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace procedural\spec;
+
+use function prodecural\echoScript;
+
+describe('procedural', function () {
+  describe('->echoScript()', function () {
+    it('echos input string inside a script tag', function () {
+      $closure = function() { echoScript('baz'); };
+      expect($closure)->toEcho('<script>baz</script>');
+    });
```

Comment:
Would be good to add a negative test assertion here. But I didn't do it yet because I can't get the tests to run due to the autoloader snag (still learning how to use Composer - I'm a total PHP n00b).